Shared/Samples/Manage features/ManageFeaturesView.swift

```diff
@@ -0,0 +1,362 @@
+        let table = data.featureTable
+        
+        do {
+            feature.damageKind = feature.damageKind?.next ?? .inaccessible
+            try await table.update(feature)
+            _ = try await table.serviceGeodatabase?.applyEdits()
+            status = "Update attribute succeeded."
+        } catch {
+            status = "Error updating attribute."
+        }
+    }
+    
+    /// Updates the geometry of a feature and applies edits to the service.
+    /// This moves the feature to the center of the map.
+    /// - Parameters:
+    ///   - feature: The feature to update.
+    ///   - geometry: The new geometry.
+    func updateGeometry(for feature: Feature, geometry: Geometry?) async {
+        guard case .success(let data) = data else { return }
+        let table = data.featureTable
+        
+        do {
+            feature.geometry = geometry
+            try await table.update(feature)
+            _ = try await table.serviceGeodatabase?.applyEdits()
+            status = "Update geometry succeeded."
+        } catch {
+            status = "Error updating geometry."
+        }
+    }
+    
+    /// Deletes a feature from the table and applies edits to the service.
+    /// - Parameter feature: The feature to delete.
+    func delete(feature: Feature) async {
+        guard case .success(let data) = data else { return }
+        let table = data.featureTable
+        
+        do {
+            try await table.delete(feature)
+            _ = try await table.serviceGeodatabase?.applyEdits()
+            status = "Delete feature succeeded."
+        } catch {
+            status = "Error deleting feature."
+        }
+    }
+}
+
+extension ManageFeaturesView {
+    /// Data value for the sample.
+    struct Data {
+        /// The map that will be displayed.
+        let map: Map
+        /// The service geodatabase.
+        let geodatabase: ServiceGeodatabase
+        /// The service feature table.
+        let featureTable: ServiceFeatureTable
+        /// The feature layer.
+        let featureLayer: FeatureLayer
+    }
+}
+
+extension ManageFeaturesView {
+    /// A value that describes the damage assessment value for a feature.
+    enum DamageKind: String, CaseIterable {
+        case inaccessible = "Inaccessible"
+        case affected = "Affected"
+        case minor = "Minor"
+        case major = "Major"
+        case destroyed = "Destroyed"
+        
+        /// The next damage kind to set on a feature.
+        var next: Self {
+            let allCases = Self.allCases
+            let index = allCases.firstIndex(of: self)!
+            let nextIndex = allCases.index(after: index)
+            return allCases[nextIndex == allCases.count ? 0 : nextIndex]
+        }
+    }
+}
+
+extension Feature {
+    /// The name of the damage type field.
+    static let damageTypeFieldName = "typdamage"
+    
+    /// The name of the object ID field.
+    static let objectIDFieldName = "objectid"
```

Comment:
Could you use the objectIDField property on the table here? Or maybe when getting the attribute below?

Reply:
Yes, I can do that.